### DISCUSSION TITLE
RenderPlugin: support relative paths for `renderFile`

### DISCUSSION THREAD

**Discussion by megheaiulian** [11ty/eleventy#4059] [Enhancement Queue]
### Is your feature request related to a problem? Please describe.

When using the RenderPlugin you have to specifiy paths relative to the project dir. 
This is very annoying when you want to put the template and the related css/js file in the same folder. 

### Describe the solution you'd like

At least for .liquid templates pass tagToken.file (the file where the tag is in) as context to shortcodeFn (https://github.com/11ty/eleventy/blob/main/src/Engines/Liquid.js#L172). 
This would allow the RenderPlugin to access `file` from context. 
When `file` is available the RenderPlugin could first try a relative path based on `path.dirname(file)`  then fallback to a project relative path and finally error out if path is not found

**Comment by zachleat** [comment]
> By default this repo closes enhancement request issues and places them in a queue for folks to upvote. We do this to avoid a large build-up of open issues. We got this idea from the [lodash project](https://twitter.com/samselikoff/status/991395669016436736).

> Don’t forget to upvote the first comment with 👍 to register your vote! You can view [the current enhancement backlog](https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+).
